src/Model/List/ArrayList.php

```diff
@@ -33,14 +32,6 @@ class ArrayList extends ModelData implements SS_List
 {
     use SearchFilterable;
 
-    /**
-     * Whether filter and exclude calls should be case sensitive by default or not.
-     * This configuration property is here for backwards compatability.
-     *
-     * @deprecated 5.1.0 use SearchFilter.default_case_sensitive instead
```

Comment:
So, is ArrayList going to use SearchFilter.default_case_sensitive ? Quick skim of the diff I cannot see it being referenced, though I may have simply missed it